src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -260,6 +260,16 @@ export const AzureStorageDetails = ({ azureContext, storageDetails }) => {
   ]);
 };
 
+export const WorkspaceInformation = ({ workspace }) => {
```

Comment:
🥳 
Smaller components!

Reply:
Yeah, I decided to push off converting the file to TS though since we pointed it as part of a future story.